src/modules/Extension/Service.php

```diff
@@ -324,32 +324,7 @@ public function findExtension($type, $id)
 
     public function update(\Model_Extension $model)
```

Comment:
Does this function do anything, other than throw an exception now? Leaving it isn't a problem, but if it doesn't do anything why not remove it?

Reply:
It's being referred to in some tests. Didn't do a search in the core, but I thought it made sense leave it there as we are going to make the function work again the future anyway.

And yeah, right now it doesn't do anything other than throwing an exception.

Reply:
Unless I missed something when I looked at it a few weeks ago, there are zero references to the function in the core. I personally would remove it entirely rather than leaving an empty function that doesn't actually do anything.